web/web.go

```diff
@@ -110,6 +111,40 @@ func (s *PrometheusStatus) ApplyConfig(conf *config.Config) bool {
 	return true
 }
 
+// PrometheusVersion contains various build information about Prometheus
+type PrometheusVersion struct {
+	Version   string
```

Comment:
Let's be explicit about the json names `Version string `json:"version"`. The current version will change the keys from lowercase to uppercase. See http://demo.robustperception.io:9090/version